suclass/cms/templates/body-post.html

```diff
@@ -1 +1,14 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<!-- 
+We have to wait to append this script to the page, otherwise there's a chance the zendesk widget does not show up on the page. Most likely, due to a race condition where other elements of the page must load first.
+-->
```

Comment:
As implemented, this is a JavaScript comment;
that means it's sent over the wire to the client/browser.

If we instead implement it as a template/python comment,
it'll be a server-side-only comment and will _not_ be sent over the wire.

It's not a _huge_ payload, but that starts to add up.
And since it's there for us as developers, server-side is fine :)

You can see a reference of these comments in `cms/templates/base.html`:

```python
## Comment goes here
## You need to double up the comment character.
```

Let's make this switch.

Reply:
*HTML Comment

Reply:
Yup, that's what my brain meant :)